trademarks.md

```diff
@@ -89,6 +89,16 @@ In those cases, you may use the word "Jupyter" or the unaltered logos to indicat
 without our prior approval.
 This is true both for non-commercial and commercial uses.
 
+Use of stills, videos, and screenshots of the Jupyter applications (including
+but not limited to: Notebook, JupyterLab, JupyterHub, nbconvert, IPython) as
+well as Jupyter websites are allowed. Any stills, videos, and screenshots of a
+specific notebook document, or any other file displayed by the application must
+be permitted by the license governing that particular document and are not
+subject to this approval. This approval is worldwide, irrevocable, and in
+perpetuity both for non-commercial and commercial uses. Any picture containing a
+person is subject to that individual's approval.
```

Comment:
Something like this is what I mean about us restricting our approval to just Jupyter trademarks, and not asserting other approvals that are out of scope here.

```suggestion
Use of stills, videos, and screenshots of the Jupyter applications (including
but not limited to: Notebook, JupyterLab, JupyterHub, nbconvert, IPython) as
well as Jupyter websites are allowed. This approval is worldwide, irrevocable, and in
perpetuity both for non-commercial and commercial uses. (Keep in mind that this approval only covers the Jupyter trademarks, and other restrictions to images or videos might apply such as content licenses or approval from a person whose likeness appears in the image.)
```

Reply:
That would be fine with me as well.